src/Generator.php

```diff
@@ -65,10 +70,11 @@ public function __construct($settings = null)
      * @param array    $categories
      * @param array    $offers
      * @param array    $deliveries
+     * @param bool     $echoOutput
      *
-     * @return bool
+     * @return bool|string
      */
-    public function generate(ShopInfo $shopInfo, array $currencies, array $categories, array $offers, array $deliveries = [])
+    public function generate(ShopInfo $shopInfo, array $currencies, array $categories, array $offers, array $deliveries = [], $echoOutput = true)
```

Comment:
`$echoOutput = true` - Default behaviour have to be `false` because of BC.

Also please describe this new option in documentation.